Components/AccordionVertical/style.styl

```diff
@@ -0,0 +1,69 @@
+$rupture.scale = lookup('$globalRuptureScale') || (0 575px 767px 991px 1199px)
+$rupture.anti-overlap = lookup('$globalRuptureAntiOverlap') || -1px
+$containerPadding = lookup('$globalContainerPadding') || 15px
+$containerMaxWidth = lookup('$globalContainerMaxWidth') || 1140px
+$colorBrand = lookup('$globalColorBrand') || #254fc7
+$buttonBackgroundColor = #f3f3f3
```

Comment:
I also think we should just define more colors directly in our base components instead of referencing global colors again (since we need to touch all of that anyway).

I'm fine with that